public/planning/poste/js/planning.js

```diff
@@ -1440,7 +1440,7 @@ function bataille_navale(poste,date,debut,fin,perso_id,barrer,ajouter,site,tout,
 
       // Ajout du widget pour copier les agents dans
       // la cellule immédiatement à droite.
-      $("#td"+cellule).append('<a class="pl-icon arrow-right" href="#"></a>');
+      $("#td"+cellule).append('<a class="pl-icon arrow-right"></a>');
```

Comment:
I would perhaps add role="button" for accessibility:
`$("#td"+cellule).append('<a class="pl-icon arrow-right" role="button"></a>');`
See https://developer.mozilla.org/fr/docs/Web/Accessibility/ARIA/Roles/button_role